<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grabber</class>
 <widget class="QWidget" name="Grabber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grabber</string>
  </property>
  <widget class="QTextBrowser" name="infoBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>471</width>
     <height>221</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="piclabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>571</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>7</number>
   </property>
   <widget class="QWidget" name="motimetab">
    <attribute name="title">
     <string>time/m/all</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="momemorytab">
    <attribute name="title">
     <string>memory/m/all</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="yetimetab">
    <attribute name="title">
     <string>time/y/all</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="yememorytab">
    <attribute name="title">
     <string>memory/y/all</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="umotimetab">
    <attribute name="title">
     <string>time/m/usr</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="umomemorytab">
    <attribute name="title">
     <string>memory/m/usr</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="uyetimetab">
    <attribute name="title">
     <string>time/y/usr</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="uyememorytab">
    <attribute name="title">
     <string>memory/y/usr</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>xPit Data Spy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>370</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>25</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>export chart</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>411</x>
     <y>59</y>
     <width>82</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="minButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>min to tray</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="logoutButton">
      <property name="text">
       <string>LOGOUT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="plotButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PLOT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>116</y>
     <width>95</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>set update interval:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="setintEdit">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>111</y>
     <width>69</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="displayFormat">
    <string>HH:mm:ss</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>112</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>set</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>10</y>
     <width>121</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>active user:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="activeusrEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="optionsButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>370</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Â© xPIT Data Spy, Quantum FX Edition V.1.0, by JA &amp; CD, 2020</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>370</y>
     <width>246</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>select date for data visualization:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="displayFormat">
       <string>MM.yyyy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
